--- debian/templates.old	Thu Jul 26 10:13:46 2001
+++ debian/templates	Thu Jul 26 10:24:43 2001
@@ -2,7 +2,7 @@
 Type: boolean
 Default: true
 Description: Do you want to create framebuffer devices?
- fbset needs a framebuffer devices to function.
- .
- May I create these device files?
+ fbset needs framebuffer devices in order to function correctly, these
+ will be created as /dev/fb[0-7], with the appropriate permissions. May I
+ create these character devices?
 
--- Makefile.old	Thu Jul 26 10:19:15 2001
+++ Makefile	Thu Jul 26 10:22:07 2001
@@ -7,6 +7,7 @@
 FLEX =		flex
 INSTALL =	install
 RM =		rm -f
+DESTDIR =
 
 ifneq (,$(findstring debug,$(DEB_BUILD_OPTIONS)))
   CFLAGS += -g
@@ -30,20 +31,22 @@
 con2fbmap.o:	con2fbmap.c
 
 install:	fbset
-		if [ -f /sbin/fbset ]; then rm /sbin/fbset; fi
-		$(INSTALL) fbset $(PREFIX)/sbin
-		$(INSTALL) fbset.8 $(PREFIX)/share/man/man8
-		$(INSTALL) fb.modes.5 $(PREFIX)/share/man/man5
-		$(INSTALL) con2fbmap $(PREFIX)/sbin
-		$(INSTALL) con2fbmap.8 $(PREFIX)/share/man/man8
-		if [ ! -c /dev/fb0 ]; then mknod /dev/fb0 c 29 0; fi
-		if [ ! -c /dev/fb1 ]; then mknod /dev/fb1 c 29 32; fi
-		if [ ! -c /dev/fb2 ]; then mknod /dev/fb2 c 29 64; fi
-		if [ ! -c /dev/fb3 ]; then mknod /dev/fb3 c 29 96; fi
-		if [ ! -c /dev/fb4 ]; then mknod /dev/fb4 c 29 128; fi
-		if [ ! -c /dev/fb5 ]; then mknod /dev/fb5 c 29 160; fi
-		if [ ! -c /dev/fb6 ]; then mknod /dev/fb6 c 29 192; fi
-		if [ ! -c /dev/fb7 ]; then mknod /dev/fb7 c 29 224; fi
+		# Frankly, let dpkg worry about that.
+		#if [ -f /sbin/fbset ]; then rm /sbin/fbset; fi
+		$(INSTALL) fbset $(DESTDIR)/sbin
+		$(INSTALL) fbset.8 $(DESTDIR)/share/man/man8
+		$(INSTALL) fb.modes.5 $(DESTDIR)/share/man/man5
+		$(INSTALL) con2fbmap $(DESTDIR)/sbin
+		$(INSTALL) con2fbmap.8 $(DESTDIR)/share/man/man8
+		# Handle this case in the postinst?
+		#if [ ! -c /dev/fb0 ]; then mknod /dev/fb0 c 29 0; fi
+		#if [ ! -c /dev/fb1 ]; then mknod /dev/fb1 c 29 32; fi
+		#if [ ! -c /dev/fb2 ]; then mknod /dev/fb2 c 29 64; fi
+		#if [ ! -c /dev/fb3 ]; then mknod /dev/fb3 c 29 96; fi
+		#if [ ! -c /dev/fb4 ]; then mknod /dev/fb4 c 29 128; fi
+		#if [ ! -c /dev/fb5 ]; then mknod /dev/fb5 c 29 160; fi
+		#if [ ! -c /dev/fb6 ]; then mknod /dev/fb6 c 29 192; fi
+		#if [ ! -c /dev/fb7 ]; then mknod /dev/fb7 c 29 224; fi
 
 clean:
 		$(RM) *.o fbset con2fbmap lex.yy.c modes.tab.c modes.tab.h
